### ISSUE TITLE
lasher test set for validation

### ISSUE THREAD

**Issue by saymamba** [dongdong2061/IJCAI25-TUMFNet#1]
Excellent work! I have a small question. I noticed that TUMFNet uses the lasher test set for validation, which is different from the previous trackers. Is this reasonable?

**Comment by dongdong2061** [comment]
Current RGBT tracking methods typically adopt the LasHeR dataset for training, and are evaluated on datasets such as the LasHeR test set, RGBT234, and RGBT210.

**Comment by saymamba** [comment]
Thank you for your quick reply! Yes, as you said, I understand that you use the LasHeR training set for training and the LasHeR test set, as well as other datasets like RGBT234 for direct inference to obtain the results. My question is that, after reading the code, I noticed that during the model training process, you used the LasHeR test set directly as the validation set, instead of sampling the LasHeR training set as the validation set.

**Comment by dongdong2061** [comment]
Yes, our method is based on BAT. Current RGBT tracking methods typically partition the LasHeR dataset into a training set and a test set, and utilize the test set for validation during training. This practice is common, as seen in other recent RGBT tracking methods.

**Comment by saymamba** [comment]
This is slightly different from what I understood, but that's okay, thank you for your work!
